scripts/seed.py

```diff
@@ -0,0 +1,210 @@
+# Command to run : python manage.py shell --settings=settings.dev  < scripts/seed.py
+
+from django.conf import settings
+from django.core.files.uploadedfile import SimpleUploadedFile
+from django.contrib.auth.models import User
+from django.utils import timezone
+
+from allauth.account.models import EmailAddress
+
+from datetime import timedelta
+from faker import Factory
+
+import os
+
+from challenges.models import Challenge, ChallengePhase, DatasetSplit, Leaderboard, ChallengePhaseSplit
+from hosts.models import ChallengeHostTeam, ChallengeHost
+from participants.models import Participant, ParticipantTeam
+
+
+def create_admin_user():
+    username = "admin"
+    password = "password"
+    email = "admin@gmail.com"
```

Comment:
Can we keep email as admin@example.com or admin@admin.com. Idea is lets not promote any mail client here and it may clash with the real one (if it exists). Also please make sure that no where `gmail` is used.